src/main/kotlin/org/malachite/estella/interview/api/InterviewController.kt

```diff
@@ -0,0 +1,64 @@
+package org.malachite.estella.interview.api
+
+import org.malachite.estella.commons.OwnResponses
+import org.malachite.estella.commons.models.interviews.InterviewNote
+import org.malachite.estella.interview.domain.InterviewId
+import org.malachite.estella.services.InterviewService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.ResponseEntity
+import org.springframework.transaction.annotation.Transactional
+import org.springframework.web.bind.annotation.*
+import java.sql.Timestamp
+import java.util.*
+import javax.sql.rowset.serial.SerialClob
+
+@RestController
+@RequestMapping("/api/interview")
+class InterviewController(
+        @Autowired private val interviewService: InterviewService
+) {
+
+    @CrossOrigin
+    @GetMapping("/jobseeker/{interviewId}")
+    fun getJobseekerName(@PathVariable interviewId: InterviewId): ResponseEntity<JobseekerName> =
```

Comment:
Jak już coś, to raczej powinno być to nazwane `InterviewIdDto`. Raczej nie potrzebujemy tego typu, bo to tak naprawdę tylko String, a jakoś bardzo DDD się nie trzymamy.

Reply:
Zwłaszcza że nie widzę definicji tego typu tutaj, więc trochę się boję, że to walnie XDD